env/src/lib.rs

```diff
@@ -283,6 +280,21 @@ impl LogBuilder {
 }
 
 impl Logger {
+    pub fn new() -> Logger {
+        let mut builder = LogBuilder::new();
+
+        if let Ok(s) = env::var("RUST_LOG") {
+            builder.parse(&s);
+        }
+
+        builder.build()
+    }
+
+    pub fn filter(&self) -> LogLevelFilter {
```

Comment:
I think of it as an instance of the usual "counting fence posts" vs "counting gaps" feature.